src/main_page/keep_or_delete.html

```diff
@@ -70,13 +72,25 @@ <h1>KeepOrDelete</h1>
    <div id="tooltip">Swipe left to delete, right to keep. You can also use arrow keys!</div>
 
    <div>
-      <a href="../trash_page.html" id="trash_button">
-         <img src="trash.jpg" alt="Clickable Image" width="50">
-      </a>
+      <img src="trash.jpg" id="trash_button" alt="Trash Icon">
    </div>
 
+   <dialog id="trash_dialog">
+      <h3>Current files to be deleted</h3>
+      <div class="trash-container">
+         <h5>Deleted Files</h5>
+         <ul id="deletedFilesList"></ul>
+         <h3 id="deletedFilesNum"></h3>
+         <button id="closeTrashModal">Close</button>
+      </div>
+
+   </dialog>
```

Comment:
```suggestion
   <dialog id="trash_dialog">
      <button id="closeTrashModal">Close</button>
      <h3>Current files to be deleted</h3>
      <div class="trash-container">
         <h5>Deleted Files</h5>
         <ul id="deletedFilesList"></ul>
         <h3 id="deletedFilesNum"></h3>
      </div>
   </dialog>
```
The button should probably be at the top so you don't have to scroll down through 100 deleted files just to close it, we can make it an X icon or something if we want in the CSS changes